libs/application-generic/src/usecases/create-workflow/create-workflow.usecase.ts

```diff
@@ -267,6 +270,7 @@ export class CreateWorkflow {
         : {}),
       ...(command.data ? { data: command.data } : {}),
     });
+    this.logger.info(`Created  workflow ${JSON.stringify(savedWorkflow)}`);
```

Comment:
I think it would be a good idea to raise if we want to log more in the engineering coffee meeting.

Reply:
```suggestion
```